wire-ios/Wire-iOS Tests/ConversationList/Container/ConversationListViewControllerSnapshotTests.swift

```diff
@@ -283,7 +283,7 @@ final class ConversationListViewControllerSnapshotTests: XCTestCase {
         // THEN
         snapshotHelper.verify(matching: renderedImage())
     }
-    
+
```

Comment:
Was this code skipped by the formatter?

Reply:
Maybe it was skipped because I rebased and not merged develop into my branch.